src/EntityFramework/Core/Mapping/MappingItemLoader.cs

```diff
@@ -2092,7 +2090,7 @@ private void ValidateFunctionImportMappingParameters(XPathNavigator nav, EdmFunc
             var lineInfo = new LineInfo(nav);
 
             // Process entity type.
-            var entityTypeString = GetAttributeValue(nav.Clone(), MslConstructs.EntitySetMappingTypeNameAttribute);
+            GetAttributeValue(nav.Clone(), MslConstructs.EntitySetMappingTypeNameAttribute);
```

Comment:
Sure I can add it - but what I really want is someone who knows this code to review the code to see whether the side-effects happen/are needed. If not I'll remove the whole line (and the comment). Who's the expert here - @ajcvickers?